ur_description/urdf/ur10.xacro

```diff
@@ -17,6 +17,6 @@
 
     Refer to 'inc/ur_macro.xacro' for more information.
   -->
-  <xacro:include filename="$(find ur_description)/urdf/inc/ur10_macro.xacro"/>
-  <xacro:ur10_robot prefix="" />
+  <xacro:arg name="model" default="10"/>
+  <xacro:include filename="$(find ur_description)/urdf/ur.xacro"/>
```

Comment:
A specific urX.xacro includes the generic ur.xacro file with the default robot_model arg changed only. All default args in ur.xacro can be changed in urX.xacro as well, since the are forwarded. Only the generic macro in ur_macro.xacro is included and called by ur.xacro.
A urX_macro.xacro might call the generic macro with robot_model only as well in an adjusted ur_macro.xacro.